tf2onnx/rewriter/cond_rewriter.py

```diff
@@ -104,28 +104,63 @@ def run(self):
 
         return self.g.get_nodes()
 
+    def _get_output_shape_dtype(self, cond_context):
```

Comment:
I actually refactored this a bit in https://github.com/onnx/tensorflow-onnx/pull/340. But you can go ahead with this, I'll merge it.